config/settings.yml

```diff
@@ -16,6 +16,8 @@
   :ldaphost:
   :ldapport: '389'
   :mode: database
+  :http_group_delimiters: ";:,"
+
```

Comment:
bonus newline here should be moreved.

Reply:
done.
also, another commit actually blanked this value out.